apps/src/templates/studioHomepages/BorderedCallToAction.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+import color from '../../util/color';
+import styleConstants from '../../styleConstants';
+import Button from '../Button';
+import {navigateToHref} from '@cdo/apps/utils';
+
+const BorderedCallToAction = props => {
```

Comment:
nit: since you're destructuring all needed values from `props` on the next line, i'd just do it in the function signature instead

```javascript
const BorderedCallToAction = ({headingText, descriptionText}) => {}
```